test_util/aws.py

```
@@ -127,11 +127,36 @@ def wait_for_stack_deletion(self, wait_before_poll_min=3):
         self.wait_for_status_change('DELETE_IN_PROGRESS', 'DELETE_COMPLETE', wait_before_poll_min)
 
     def get_parameter(self, param):
+        """Returns param if in stack parameters, else returns None
```

Will switch to fail hard if not found